express_server/server.js

```diff
@@ -19,12 +19,12 @@ app.post("/askgpt", async (req, res) => {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
-        "Authorization": `Bearer ${OPENAI_API_KEY}`,
+        Authorization: `Bearer ${OPENAI_API_KEY}`,
```

Comment:
Whats the reason behind this change?

Reply:
Content-Type has a dash, and hence it needs to be wrapped :^) ""